src/app/main.py

```diff
@@ -1,19 +1,20 @@
 from fastapi import FastAPI
-import requests as req
+from app.core.config import settings
+from app.api.api_v1.api import api_router
+from app.db.init_db import init_db
+
+app = FastAPI(
```

Comment:
For the sake of not indicating a closer grouping of imports with this line than with the top level functions here we should have two newlines between imports and this statement